src/test/rgw/bucket_notification/setup.py

```diff
@@ -15,5 +15,7 @@
     install_requires=[
         'boto >=2.0b4',
         'boto3 >=1.0.0'
+        'cloudevents',
```

Comment:
I think for installation this should be written in `src/test/rgw/bucket_notification/requirements.txt`.

Reply:
thanks. it works!
BTW, do you know why we have setup.py ?

Reply:
That's great!
Basically what I think is `setuptools` uses "easy_install" and here we use "pip" for installation of packages. So `install_requires` section takes care that the packages are installed properly and it will fail if they're not installed.